<dec f='boost-1.85.0/libs/log/include/boost/log/exceptions.hpp' l='433' type='void boost::log::limitation_error::limitation_error()'/>
<doc f='boost-1.85.0/libs/log/include/boost/log/exceptions.hpp' l='430'>/*!
     * Default constructor. Creates an exception with the default error message.
     */</doc>
<def f='boost-1.85.0/libs/log/src/exceptions.cpp' l='600' ll='603' type='void boost::log::limitation_error::limitation_error()'/>
<use f='boost-1.85.0/libs/log/src/exceptions.cpp' l='616' u='c' c='_ZN5boost3log12v2s_mt_posix16limitation_error6throw_EPKcm'/>
